client/modules/workspace/src/JobsDetailModal/JobsDetailModal.tsx

```diff
@@ -1,16 +1,192 @@
+    [lastMessageTitle]: (
+      <Collapse
+        defaultActiveKey={[]}
+        expandIconPosition="end"
+        size="small"
+        bordered={false}
+        style={{ background: '#fff' }}
+        items={[
+          {
+            label: (
+              <span>
+                {hasFailedStatus ? 'Last Status Message' : 'System Output'}
+              </span>
+            ),
+            children: <pre>{jobData.lastMessage}</pre>,
+          },
+        ]}
+      ></Collapse>
+    ),
+    ...(jobData.remoteOutcome && { 'Remote Outcome': jobData.remoteOutcome }),
+  };
+
+  if (jobData.remoteOutcome) {
+    statusDataObj['Remote Outcome'] = jobData.remoteOutcome;
+  }
+
+  const configDataObj = {
+    'Execution System': jobDisplay.systemName,
+    'Max Minutes': String(jobData.maxMinutes),
+    queue: 'queue' in jobDisplay ? jobDisplay.queue : undefined,
+    'Cores On Each Node':
+      'coresPerNode' in jobDisplay
+        ? String(jobDisplay.coresPerNode)
+        : undefined,
+    'Node Count':
+      'coresPerNode' in jobDisplay
+        ? String(jobDisplay.coresPerNode)
+        : undefined,
+    Allocation: 'allocation' in jobDisplay ? jobDisplay.allocation : undefined,
+    'Execution Directory':
+      'execSystemExecDir' in jobData ? jobData.execSystemExecDir : undefined,
+  };
+  const outputDataObj = {
+    'Job Name': jobData.name,
+    'Output Location': outputLocation,
+    'Archive System': jobData.archiveSystemId,
+    'Archive Directory': jobData.archiveSystemDir,
+  };
+
+  const inputDataObj = {
+    ...reduceInputParameters(jobDisplay.inputs),
+  };
+
+  const paramsDataObj = {
+    ...reduceInputParameters(jobDisplay.appArgs),
+    ...reduceInputParameters(jobDisplay.envVars),
+  };
+
+  const valueStyle = {
+    paddingLeft: '40px',
+    paddingBottom: '0px',
+    width: '100%',
+  };
+
+  const labelStyle = {
+    color: '#484848',
+    fontWeight: 700,
+    paddingTop: '10px',
+    border: '0',
+  };
+
+  const baseRowStyle = {
+    paddingLeft: '12px',
+    paddingRight: '12px',
+    paddingBottom: '5px',
+  };
+
+  const itemLabelStyle = {
+    color: 'rgba(0, 0, 0, 0.88)',
+    font: 'normal normal 14px Helvetica Neue',
+    paddingBottom: '0px',
+  };
+
+  interface DisplayItemsProps {
+    items: DescriptionsProps['items'];
+  }
+
+  const DisplayItems: React.FC<DisplayItemsProps> = ({ items }) => (
+    <Descriptions
+      size="small"
+      bordered={false}
+      contentStyle={{ paddingBottom: '0px' }}
+      labelStyle={itemLabelStyle}
+      style={{ paddingLeft: '40px', paddingBottom: 0 }}
+      layout="vertical"
+      column={1}
+      items={items}
+    />
+  );
+
+  const getItems = (
+    data: Record<string, string | undefined>
+  ): DescriptionsProps['items'] =>
+    Object.entries(data)
+      .filter(([_, value]) => value)
+      .map(([key, value]) => ({
+        label: key,
+        children: <span style={valueStyle}>{value}</span>,
+      }));
+
+  const dataObjects = [
```

Comment:
Should we hide sections that have no data?
<img width="751" alt="Screenshot 2024-07-10 at 2 07 57 PM" src="https://github.com/DesignSafe-CI/portal/assets/20326896/b8c410ce-db2b-4e9b-bd87-9f1b36170ee0">

Reply:
Sure, I can hide it, this will follow what we do in app form detail.
Right now, I choose to keep to be in sync core portal job details.

Reply:
Fixed it and tested.
Example for matlab with no inputs and parameters

<img width="1032" alt="Screenshot 2024-07-10 at 2 33 46 PM" src="https://github.com/DesignSafe-CI/portal/assets/2568355/ab9f23bd-3ef9-4a50-9802-afbd781a4a39">